src/protocols/BaseProtocolAdapter.ts

```diff
@@ -77,8 +77,8 @@ export abstract class BaseProtocolAdapter {
 
   abstract fetchTransactions(
     address: string,
-    lastTxId?: string,
-  ): Promise<any>;
+    nextPageParams: string | null,
```

Comment:
I suggest to use `undefined` instead of `null` so this can by typed with the use of question mark.

Reply:
null is returned from api when we get the latest page

Reply:
OK